hw_5/point_hw5.py

```diff
@@ -0,0 +1,29 @@
+class Circle:
+    def __init__(self, x=0, y=0, radius=10):
+        self.x = x
+        self.y = y
+        self.radius = radius
+        print(self.x, self.y, self.radius)
+
+    def in_circle(self, f):
```

Comment:
rename f variable with some meaningful name, like point, and its a good idea to add an annotation.

Reply:
fix